workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css

```diff
@@ -302,39 +304,9 @@ pre[class*='language-'] {
 		}
 	}
 
-	.learn-article-page-nav {
-		> ul {
-			align-items: flex-start;
-			display: flex;
-			flex-direction: column;
-			gap: var(--spacer-2);
-		}
-
-		> li {
-			align-items: center;
-			margin-left: 10px;
-			min-width: 100%;
-
-			.active {
-				border-left: 4px solid var(--color-brand-primary, #0b5fff);
-				color: var(--color-neutral-10, #282934) !important;
-				padding: 0 0 0 6px !important;
-			}
-
-			> a {
-				color: var(--color-neutral-6, #82828c) !important;
-				font-family: Source Sans 3;
-				font-size: 1rem;
-				font-style: normal;
-				font-weight: 600;
-				line-height: 1.5rem;
-				padding: 0 0 0 10px !important;
-
-				&:hover {
-					color: var(--color-neutral-10, #282934) !important;
-				}
-			}
-		}
+	.learn-article-wrapper {
```

Comment:
@rtienpal I dont think you did this, but can you make sure the .info-bar is correctly alphabetized? it looks like its not in the correct order

Reply:
I just sent new commit with the adjustment, although I searched throughout Learn's site and didn't find this element, maybe we should delete this css class?